go/obscuronode/node.go

```diff
@@ -229,13 +237,96 @@ func (a *Node) Stop() {
 	a.exitNodeCh <- true
 }
 
+func sendInterrupt(interrupt *int32) *int32 {
```

Comment:
just a general comment for the future.
It's better to not move functions around the file, as it makes the review much more painful

Reply:
Apologies for that, this PR kinda went a bit longer that I initially expected 😅